plugins/inputs/procstat/procstat_linux_test.go

```diff
@@ -0,0 +1,624 @@
+package procstat
+
+import (
+	"net"
+	"testing"
+	"time"
+
+	"github.com/elastic/gosigar/sys/linux"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
```

Comment:
Please use the `require` calls instead of `assert` and group the imports

```suggestion
	"github.com/elastic/gosigar/sys/linux"
	"github.com/stretchr/testify/require"

	"github.com/influxdata/telegraf"
	"github.com/influxdata/telegraf/testutil"
```